src/main/java/frc/trigon/robot/commands/factories/ShootingCommands.java

```diff
@@ -81,6 +81,17 @@ private static Command getFeedNoteForCloseSpeakerShotCommand() {
         return GeneralCommands.runWhen(IntakeCommands.getSetTargetStateCommand(IntakeConstants.IntakeState.FEED_SHOOTING).alongWith(getVisualizeNoteShootingCommand()), () -> RobotContainer.SHOOTER.atTargetVelocity() && RobotContainer.PITCHER.atTargetPitch());
     }
 
+    public static Command getShootManualLowDeliveryCommand() {
+        return new ParallelCommandGroup(
```

Comment:
Maybe put this above since it's public?

Reply:
I have it organized by shooting type. The normal shooting is all on the top, then close shot stuff, and then manual delivery.

Reply:
I didn't notice that at first, and missed some commands. 
1. You should make the prepare command private
2. Maybe a more readble way would be to put the public commands at the top, and then similar commands next to each other. Right now, I've missed some public commands.